src/components/group-common.js

```diff
@@ -38,37 +39,65 @@ export const AdminNotifierCheckbox = ({ checked, onChange }) => {
 }
 
 export const NewUserCard = ({ onClick }) => {
-  return h(Clickable, {
-    style: Style.cardList.shortCreateCard,
+  return h(ButtonPrimary, {
+    style: { textTransform: 'none' },
     onClick
   }, [
-    div(['Add a User']),
-    icon('plus-circle', { style: { marginTop: '0.5rem' }, size: 21 })
+    icon('plus', { size: 14 }),
+    div({ style: { marginLeft: '0.5rem' } }, ['Add User'])
   ])
 }
 
+const UserMenuContent = ({ onEdit, onDelete }) => {
+  return h(Fragment, [
+    h(MenuButton, {
+      onClick: () => onEdit()
```

Comment:
These shouldn't be wrapped in functions:
```js
{ onClick: onEdit }
```

Reply:
We appear to have a lack of consistency in our codebase on this, I was following the pattern here: https://github.com/DataBiosphere/terra-ui/blob/dev/src/pages/workspaces/List.js#L66 which seems like the most equivalent thing.

Should I do a comprehensive pass of the codebase and change in all places?

Reply:
That would be nifty. Some context:

Back in the days of class components, it was necessary to wrap callbacks in arrow functions, in order to ensure that the value of `this` was preserved. In the arrow function world, we don't use `this` (indeed, there is no `this` in an arrow function).

Reply:
Ah, that makes sense. I was wondering how we got around the this binding

Reply:
@zarsky-broad done, ptal. I did not do comprehensive testing, instead relying on tests and review here. If you think it would be worth going through and testing all of the refactors, I can do that.

FWIW, in the future, for this sort of refactor (especially given how long it took), it would be extremely helpful to be given context. Why did we decide to move towards direct bindings, rather than arrow functions? What benefits do we gain? What are the cons of this approach? I think that generally providing more of that information will help people understand the decisions behind the Terra UI codebase, and the code style recommendations.